app_bikes/apps.py

```diff
@@ -0,0 +1,11 @@
+# coding: UTF-8
+
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _L
```

Comment:
some old habits from another framework :) _ -> ugettext, _L -> ugettext_lazy, just to know which one you're using